serverless/serverless-config-HA.adoc

```diff
@@ -1,12 +1,12 @@
 include::modules/serverless-document-attributes.adoc[]
 [id="serverless-config-HA"]
-= Configuring high availability {ServerlessProductName} components
+= Configuring high-availability {ServerlessProductName} components
 include::modules/common-attributes.adoc[]
 :context: serverless-config-HA
 
 toc::[]
 
-This guide provides information about which components of {ServerlessProductName} are configured as high availability (HA) by default, and how you can modify HA settings.
+This guide provides information about which components of {ServerlessProductName} are configured as high-availability (HA) by default, and how you can modify HA settings.
```

Comment:
s/high-availability/high availability/
In this case, you don't need a hyphen in `high availability` as it is not an adjective modifying a noun (e.g., high-availability component).

Reply:
Ahhh got you, thanks